.pre-commit-config.yaml

```diff
@@ -7,11 +7,11 @@ repos:
       - id: typos
   - repo: local
     hooks:
-      - id: ensure Swift not changed to local development
-        name: ensure Swift not changed to local development
+      - id: ensure Swift setup for local development
```

Comment:
Now we DO want to default to local dev, and part of releasing I switch to false...